test/vscode-ui/test/specs/nested-fastapi.spec.ts

```diff
@@ -157,16 +159,21 @@ describe("Nested Fast API Deployment", () => {
 
   it("can check config", async () => {
     const workbench = await browser.getWorkbench();
-    await expect(
-      await workbench.getEditorView().getOpenEditorTitles(),
-    ).toContain("configuration-1.toml");
+    const openEditorTitles = await workbench
+      .getEditorView()
+      .getOpenEditorTitles();
+    const fileNamePattern = /^my fastapi app-.*$/;
+    const realFilename = openEditorTitles.find((title) =>
+      fileNamePattern.test(title),
+    );
```

Comment:
Should we pull this out into a testing utility function to avoid the duplicate code?